ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java

```diff
@@ -139,7 +138,7 @@ public boolean saveEvent(Event evt) {
         try {
             jedis = getJedis();
             long timeStamp = evt.getTimestamp();
-            String hashId = this.getHashKey(evt.getTimestamp(), evt.getAction());
+            String hashId = this.getHashKey(evt.getTimestamp());
             evt.setUuid(String.valueOf(timeStamp));
             jedis.zadd(hashId, timeStamp, objectMapper.writeValueAsString(evt));
             return true;
```

Comment:
why ?

Reply:
Because it was not really used in the hashKey